kolibri/core/content/utils/channel_import.py

```diff
@@ -647,13 +654,11 @@ def generate_data_with_default(record):
                         )
                     )
                 else:
-                    execute_values(
-                        cursor,
-                        # We want to overwrite new values that we are inserting here, so we use an ON CONFLICT DO UPDATE here
-                        # for the resulting SET statement, we generate a statement for each column we are trying to update
-                        "INSERT INTO {table} AS SOURCE ({column_names}) VALUES %s ON CONFLICT ({pk_name}) DO UPDATE SET {set_statement};".format(
+                    cursor.executemany(
```

Comment:
This will cause a considerable slowdown in insert performance https://stackoverflow.com/a/39034789 so I'd like us to find another way to handle this.

Reply:
Changed it using a similar implementation of `execute_values`  but ensuring that utf-8 chars are inserted in the database and not bytes.